packages/story-editor/src/components/secondaryPopup/index.js

```diff
@@ -46,13 +46,17 @@ const transitionStyles = {
 const Controller = styled.div`
   position: absolute;
   top: -12px;
-  left: 0px;
+  ${({ zIndex }) => (zIndex ? `z-index: ${zIndex};` : '')}
```

Comment:
I had to set a higher z-index here (didn't want to change z-indexes in Page Side Menu) in order to prevent this:

https://user-images.githubusercontent.com/21036224/157312306-b299a49e-8688-4822-9b83-754525af740b.mov